core/src/apps/wallet/sign_tx/addresses.py

```diff
@@ -120,6 +120,11 @@ def address_pkh(pubkey: bytes, coin: CoinInfo) -> str:
     return base58.encode_check(bytes(s), coin.b58_hash)
 
 
+def address_pkh_from_keyid(keyid: bytes, coin: CoinInfo) -> str:
```

Comment:
Please move this to `_address_pkh_from_keyid` in `src/apps/wallet/sign_tx/dash.py`

Reply:
Fixed. I've placed it as the separate commit for review, I'll squash it later if it is needed.